.eslintcache

```diff
@@ -0,0 +1 @@
0, canvas.width, canvas.height);\n}\n\nfunction createImageData(canvas: HTMLCanvasElement) {\n  var ctx = canvas.getContext('2d');\n  return ctx && ctx.createImageData(canvas.width, canvas.height);\n}\n\n","/Users/maxpinheiro/Documents/a-eye/src/modules/stateSpaces/kmeans/chartjs-plugin-dragdata.d.ts",[],"/Users/maxpinheiro/Documents/a-eye/src/modules/stateSpaces/kmeans/utils.ts",[],{"ruleId":"156","replacedBy":"157"},{"ruleId":"158","replacedBy":"159"},{"ruleId":"156","replacedBy":"157"},{"ruleId":"158","replacedBy":"159"},{"ruleId":"160","severity":1,"message":"161","line":14,"column":11,"nodeType":"162","messageId":"163","endLine":14,"endColumn":24},{"ruleId":"164","severity":1,"message":"165","line":8,"column":61,"nodeType":"162","messageId":"166","endLine":8,"endColumn":73},{"ruleId":"164","severity":1,"message":"167","line":114,"column":10,"nodeType":"162","messageId":"166","endLine":114,"endColumn":15},{"ruleId":"160","severity":1,"message":"168","line":113,"column":14,"nodeType":"162","messageId":"163","endLine":113,"endColumn":15},{"ruleId":"169","severity":1,"message":"170","line":6,"column":9,"nodeType":"171","endLine":6,"endColumn":11,"suggestions":"172"},{"ruleId":"173","severity":1,"message":"174","line":9,"column":9,"nodeType":"175","endLine":13,"endColumn":11},{"ruleId":"164","severity":1,"message":"176","line":4,"column":21,"nodeType":"162","messageId":"166","endLine":4,"endColumn":36},{"ruleId":"164","severity":1,"message":"177","line":6,"column":7,"nodeType":"162","messageId":"166","endLine":6,"endColumn":17},{"ruleId":"178","severity":1,"message":"179","line":39,"column":59,"nodeType":"180","messageId":"181","endLine":39,"endColumn":61},{"ruleId":"160","severity":1,"message":"182","line":79,"column":14,"nodeType":"162","messageId":"163","endLine":79,"endColumn":15},"no-native-reassign",["183"],"no-negated-in-lhs",["184"],"@typescript-eslint/no-redeclare","'ModuleSection' is already defined.","Identifier","redeclared","@typescript-eslint/no-unused-vars","'AxisSelector' is defined but never used.","unusedVar","'getBg' is defined but never used.","'i' is already defined.","react-hooks/exhaustive-deps","React Hook useEffect has a missing dependency: 'props'. Either include it or remove the dependency array. However, 'props' will change when *any* prop changes, so the preferred fix is to destructure the 'props' object outside of the useEffect call and refer to those specific props inside useEffect.","ArrayExpression",["185"],"jsx-a11y/alt-text","img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.","JSXOpeningElement","'createImageData' is defined but never used.","'pixelmatch' is assigned a value but never used.","eqeqeq","Expected '===' and instead saw '=='.","BinaryExpression","unexpected","'y' is already defined.","no-global-assign","no-unsafe-negation",{"desc":"186","fix":"187"},"Update the dependencies array to be: [props]",{"range":"188","text":"189"},[261,263],"[props]"]
```

Comment:
Remember that we need to remove this file entirely from the repo, to do this:
`rm .eslintcache` in the terminal and then commit the deletion of the whole file rather than just the line!